faster_whisper/transcribe.py

```diff
@@ -142,7 +146,25 @@ def __init__(
                 "openai/whisper-tiny" + ("" if self.model.is_multilingual else ".en")
             )
 
-        self.feature_extractor = FeatureExtractor()
+        feature_extractor_file = os.path.join(model_path, "preprocessor_config.json")
```

Comment:
Maybe move that into a specific method ?
And use the n_mels from ct2 as a fallback ?

Reply:
I'll try to get to this tomorrow. extremely limited bandwidth with American holidays currently